scripts/check-polish-scope.sh

```
@@ -0,0 +1,255 @@
+                echo "Error: --repo-root requires a path argument" >&2
+                exit 2
+            fi
+            REPO_ROOT="$2"
+            shift 2
+            ;;
+        --state-file)
+            if [[ -z "${2:-}" ]]; then
+                echo "Error: --state-file requires a path argument" >&2
+                exit 2
+            fi
+            STATE_FILE="$2"
```

Shellcheck (0.11.0)

[warning] STATE_FILE appears unused. Verify use (or export if used externally).

(SC2034)